### PR TITLE
docs: adds SVM details to enhancement blog post

### PR DESCRIPTION
**Pull request by nilekhc**
Add SIG API Machinery enhancements - Storage Version Migration in v1.30 release blog

/assign @enj @deads2k @jpbetz

### CODE REVIEW

**Comment by sftim** [comment]
/hold

This should get checked by Release Comms, and we should also add a (brief) note about any post-publication updates.

If you want a post-release blog article @nilekhc you are weeks beyond the deadline to request one, but you could still ask for an exception.

File: content/en/blog/_posts/2024-04-17-kubernetes-1.30.md [hunk 1]
```diff
@@ -252,6 +252,16 @@ strategy.
 See [Traffic Distribution](/docs/reference/networking/virtual-ips/#traffic-distribution) for more 
 details. 
 
+### Storage Version Migration ([SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery))
+Kubernetes v1.30 introduce new built-in API for _StorageVersionMigration_.
```
Comment (sftim) [inline content/en/blog/_posts/2024-04-17-kubernetes-1.30.md:256]:
```suggestion
Kubernetes v1.30 introduces a new built-in API for _StorageVersionMigration_.
```

File: content/en/blog/_posts/2024-04-17-kubernetes-1.30.md
Comment (sftim) [inline content/en/blog/_posts/2024-04-17-kubernetes-1.30.md:1]: - mention that the new API is alpha, and that it was previously available out of tree

**Review by jpbetz — COMMENTED** [review]
LGTM technically

/approve

**Comment by kcmartin** [comment]
Release Comms, in consultation with the Release Lead @katcosgrove, are okay with this post-publication addition, pending Kat's review, and with the addition of a note to the end of the article mentioning the post-publication change.

**Pull request merged**